rsfbclient-native/build.rs

```diff
@@ -28,6 +34,26 @@ fn search_on_linux() {
     println!("cargo:rustc-link-lib=dylib=fbclient");
 }
 
+#[cfg(all(feature = "linking", target_os = "macos"))]
+fn search_on_macos() {
+    let def_fbclient_sys = "/usr/local/lib/libfbclient.dylib";
+    let fb3_lib_path_sys = std::path::Path::new(def_fbclient_sys);
+    if fb3_lib_path_sys.exists() {
+        println!("cargo:rustc-link-search=/usr/local/lib/");
+    }
+    let def_fbclient_lib =
+        "/Library/Frameworks/Firebird.framework/Versions/A/Libraries/libfbclient.dylib";
+    let fb3_lib_path_lib = std::path::Path::new(def_fbclient_lib);
+    if fb3_lib_path_lib.exists() {
+        println!(
+            "cargo:rustc-link-search=/Library/Frameworks/Firebird.framework/Versions/A/Libraries/"
+        );
+    }
+    println!("cargo:rustc-link-lib=dylib=libfbclient");
```

Comment:
```suggestion
    println!("cargo:rustc-link-lib=dylib=fbclient");
```

Reply:
I'm unsure about this change because:
1. `cargo build` succeeds. 
2. `cargo test` fails with the same arguments.
3. The steps are in sequence. 

I have thinking the following hipotesis:
1. If `libfbclient` is wrong, `cargo build` may have failed before.
2. For some reason `cargo test` is not getting the search path of build. rs

What I am missing?

Reply:
Also this PR has the `cargo test` of the feature `linking` on `macos-latest` disabled for not breaking the build.
So we can solve this later.

Reply:
Build passes because only when building a executable (tests) the compiler call the linker, which passes the compiler flag `-llibrsfbclient` with `cargo:rustc-link-lib=dylib=libfbclient`. 
However, the linux version with `cargo:rustc-link-lib=dylib=fbclient` calls the linker with `-lfbclient` and works, and testing with `cargo:rustc-link-lib=dylib=libfbclient`, it fails with the same error as the current commit of this PR. (`cannot find -llibfbclient`)

Reply:
We can call the cargo build with the `--all-targets` flag to make sure the tests and examples are built, allowing the linker to be called without needing to run the tests.

Reply:
I will approve the pr to fix this later then.